src/preload.ts

```diff
@@ -23,36 +24,19 @@ const isMac = os.platform() === 'darwin'
 const isWindows = os.platform() === 'win32'
 const isLinux = os.platform() === 'linux'
 
-let fsWatchListeners = new Map<
-  string,
-  {
-    watcher: fsSync.FSWatcher
-    callback: (eventType: string, path: string) => void
-  }
->()
+let fsWatchListeners = new Map<string, ReturnType<typeof chokidar.watch>>()
 
-const watchFileOn = (
-  path: string,
-  callback: (eventType: string, path: string) => void
-) => {
-  const watcher = fsSync.watch(path)
-  watcher.on('change', callback)
-  fsWatchListeners.set(path, { watcher, callback })
+const watchFileOn = (path: string, callback: (path: string) => void) => {
+  const watcher = chokidar.watch(path)
```

Comment:
Should we do a check to see if the `fsWatchListeners` already has this path in some edge case so it does not do `chokidar.watch` twice on something?

Reply:
Wouldn't hurt! Even though I can't think of a case when, never a bad idea to be safer.

Reply:
Done